monkey/monkey_island/cc/resources/representations.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
```

Comment:
This is not a resource, it's a service. Move this file to services.

Reply:
Fixed in bcf2d964